api/src/main/java/org/hawkular/inventory/api/MultipleEntityBrowser.java

```diff
@@ -23,7 +23,7 @@
  * @author Lukas Krejci
  * @since 1.0
  */
-interface MultipleEntityBrowser<Entity> {
+public interface MultipleEntityBrowser<Entity> {
```

Comment:
Good, so let's go with that then.

Could you rename the two interfaces and make them public, too? I'd merge this PR once that's done.